### PR TITLE
Upgrade packages to latest

### PR DESCRIPTION
**Pull request by markhougaard**
@JG1995 Please pull this branch and click around extensively to see if something's amiss. I didn't find anything that broke, despite multiple upgrades across all packages, and that makes me worried I missed something.

### CODE REVIEW

**Review by JG1995 — APPROVED** [review]
Been clicking around and trying out various things that could potentially break for a solid 30 minutes, everything seems to work just fine. Maybe we're just lucky this time.

GitHub is complaining about conflicts that need to be resolved though, but I assume you're going to take care of that anyways?

File: package.json [hunk 1]
```diff
@@ -4,17 +4,18 @@
     "description": "",
     "main": "_site/index.html",
     "devDependencies": {
-        "@11ty/eleventy": "^0.12.1",
-        "@11ty/eleventy-navigation": "^0.2.0",
-        "@11ty/eleventy-plugin-syntaxhighlight": "^3.2.2",
+        "@11ty/eleventy": "^1.0.0",
+        "@11ty/eleventy-navigation": "^0.3.2",
+        "@11ty/eleventy-plugin-syntaxhighlight": "^3.1.1",
+
```
Comment (markhougaard) [inline package.json:10]:
```suggestion

```

File: package.json [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
         "@mapsindoors/midt": "^15.0.0",
```
Comment (markhougaard) [inline package.json:10]:
```suggestion
        "@11ty/eleventy-plugin-syntaxhighlight": "^3.1.1",
        "@mapsindoors/midt": "^15.0.0",
```

**Comment by markhougaard** [comment]
@JG1995 Conflicts are resolved—wanna give it a two-minute go again before I merge it in?

**Review by JG1995 — APPROVED** [review]
Everything appears to be in working order, so I say go for it!

**Pull request merged**